includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1,6 +1,8 @@
 <?php
 /**
  * Stripe Payment Request API
+ * Adds support for Apple Pay and Chrome Payment Request API buttons.
+ * Utilizes the Stripe Payment Request Button to support checkout from the product detail and cart pages.
```

Comment:
I added this comment here because it wasn't immediately clear why we had this class.